docs/installing-vyper.rst

```diff
@@ -76,4 +76,30 @@ Each tagged version of vyper is uploaded to `pypi <https://pypi.org/project/vype
 To install a specific version use:
 ::
 
-    pip install vyper==0.1.0b17
+    pip install vyper==0.2.15
+
+
+nix
+***
+
+View the versions supported through nix at `nix package search <https://search.nixos.org/packages?channel=21.05&show=vyper&from=0&size=50&sort=relevance&query=vyper>`_ 
+
+
+Supported platforms include:
+
+    - aarch64-linux
```

Comment:
The formatting on this looks .. not what was intended.
![Screenshot from 2021-08-19 08-33-29](https://user-images.githubusercontent.com/3867501/130097849-e6101a85-1436-4ce4-995a-c37cd9b7c4ae.png)

I think for .rst, the syntax for a bulleted list is different.

Reply:
@sambacha I think though that, since the supported platforms and the link to the derivation is already included on the nix package search (which will presumably be kept more up to date than the vyper docs), we can skip everything from "Supported platforms..." through "Installing Vyper".
![Screenshot from 2021-08-19 08-36-08](https://user-images.githubusercontent.com/3867501/130098378-5188b304-7541-4766-a022-126ac94ac477.png)